.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,36 @@
+name: Bug Report
+description: Report any bugs you encounter.
```

Comment:
Can we perhaps improve this to something like "Please enter the details of your bug" or the like